src/toolbar/README.md

```diff
@@ -0,0 +1,43 @@
+# @dojo/widgets/toolbar/Toolbar widget
+
+Dojo 2's `Toolbar` component can be used to display an application navigation bar that can show a title and action items. Action elements can be configured to automatically collapse into a `SlidePane` component, and the `Toolbar` itself can be configured to either be fixed or statically positioned.
+
+## Features
+
+- Capable of auto-collapsing action elements into `SlidePane`
+- Capable of rendering either a fixed or statically positioned root element
+- Customizable `title` element
+
+### Accessibility Features
+
+- The menu button displays screen-readable text that's visually hidden and its icon has an `aria-hidden` attribute
```

Comment:
Thinking about the menu wrapping, it might also be good to add some text like this:
> Toolbar makes no assumptions about what content will be inserted through `actions`. If you wish to create a site-wide navigation section, it would be best to wrap links in an element with the appropriate semantics (e.g. `<nav>` or `role="navigation"`)